azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/state/CallingState.kt

```diff
@@ -34,6 +34,8 @@ internal data class CallingState(
     val isTranscribing: Boolean = false,
     // set once for the duration of the call in the CallStateReducer when call start requested.
     val callStartDateTime: OffsetDateTime? = null,
+    val callEndReasonCode: Int? = null,
```

Comment:
Is there appropriate default values instead of null? Maybe 0? Maybe MAX_INT?

Reply:
As per discussion we will fix for GA.